DeferredDocumentQuery/src/main/resources/wsdl/NhinDeferredResultsOption.wsdl

```diff
@@ -0,0 +1,51 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- 
+    Deferred Results Option WSDL
```

Comment:
Remove one of the  wsdl  either this NhinDeferredResultsOption.wsdl or NhinDeferredResultsOptionResponse.wsdl. Also use soapAction urn:ihe:iti:2007:CrossGatewayQueryDeferredResults